tests/main/preseed-snapd-snap/task.yaml

```
@@ -0,0 +1,108 @@
+summary: Check that preseeding of current ubuntu cloud image works with snapd
+  and core18 works.
```

Something wonky here. Also the summary should probably not wrap like that without: `|`

```suggestion
  and core18.
```